packages/cardano-services/test/util/validators.test.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  CACHE_TTL_LOWER_LIMIT,
+  CACHE_TTL_UPPER_LIMIT,
+  MissingProgramOption,
+  ProgramOptionDescriptions,
+  ServiceNames
+} from '../../src';
+import { cacheTtlValidator } from '../../src/util/validators';
+
+describe('utils/validators', () => {
+  describe('cacheTtlValidator', () => {
+    it('returns ttl in seconds with a valid value within the range', async () => {
```

Comment:
```suggestion
    it('returns TTL in seconds with a valid value within the range', async () => {
```
and the other descriptions

Reply:
It's done, force-pushed.